components/brave_wallet/renderer/brave_wallet_js_handler.cc

```diff
@@ -107,6 +175,9 @@ v8::Local<v8::Promise> BraveWalletJSHandler::Request(
   if (!out || !out->is_dict() || !out->GetAsDictionary(&out_dict))
     return v8::Local<v8::Promise>();
 
+  // Hardcode id to 1 as it is unused
+  ALLOW_UNUSED_LOCAL(out_dict->SetIntPath("id", kRequestId));
```

Comment:
weird that these were needed, I don't see them used in other places, but I guess it's ok because other places also ignore the result